Hmwk_#3_Hangman/hangman_game.py

```diff
@@ -0,0 +1,159 @@
+import functools
+import random
+from functools import reduce
+import time
+import hangman_graphics
+
+#get data from file but not read so later it can be accesed twice without reopening
+def get_words(filename):
+    text = open(filename, "r", encoding="utf-8")
+
+    return text
+
+#define mode PC vs. PC, Player vs. PC
+def set_game_mode():
+    game_mode = input("What type of game you wanna play? (Manual / Auto) ").lower()
+    
+    if game_mode == "manual":
+        print("Manual game mode requires to interact with PC !")
+    elif game_mode == "auto":
+        print("Auto game mode allows just to watch the flow")
+
```

Comment:
It'd be better to check whether the user writes something different besides "manual" and "auto" in order to prevent program crashing.